### DISCUSSION TITLE
Excessive data in divison layer & possible feedback

### DISCUSSION THREAD

**Discussion by Fylax** [OvertureMaps/data#153] [Discussion]
Hello,

I am currently using the first beta of OvertureMaps and have noticed some data that looks out of place.

In the division layer, we've noticed the presence of *squares* that we suspect may not belong in this layer. 
Should we mark these squares as **to be deleted**?

Moreover, we're based in Italy and we are able to fix inconsistent data. We are very willingly to contribute in improving data quality.

How can provide feedback with the correct info?

**Comment by jwass** [comment]
Hi @Fylax - Thanks for looking into the data. Can you provide some examples of the data you're referring to in the divisions theme? Providing the IDs would help us sync up to start taking a look.

You can always inspect the "sources" field of an Oveture element to find where it's coming from. A lot (but not all) of the divisions data comes from OpenStreetMap so that would be a good place to provide immediate fixes to discovered issues.

**Comment by Fylax** [comment]
Hi @jwass 

Some examples are:
085ed5babfffffff0169f8a7c1fa1517 -> OSM R12354468 (https://www.openstreetmap.org/relation/12354468)
08503358bfffffff019c6325f07759a7 -> OSM W1042254232 (https://www.openstreetmap.org/way/1042254232)

I can see from OpenStreetMap that the data is marked as "place: square" and that it lacks of hierarchy.

I am not really sure if this type of places should be included in division layer (and I doubt it should have "neighborhood" subtype in OvertureMaps)

I will in any case fix data in OSM to ensure that it is properly propagated in Overture Maps

**Comment by Fylax** [comment]
Upon further investigation I found this entity: 
085f291dbfffffff01071aec90b68b25 -> OSM W494692649

Just like the ones provided in the previous post it is a square (so definitely not a "neighborhood") having not "relationships" in OSM.
Compared to the previous, this data has right hierarchies in Overture Maps.

How can I help improve OvertureMaps considering that the two examples seems to be equal on OSM?
